tests/return64bit/testbind.js

```diff
@@ -14,7 +14,7 @@ Module['runtest'] = function() {
   console.log("high = " + high);
 
   var ptr = _get_func_ptr();
-  low = dynCall_jj(ptr, 0x12345678, 0xabcdef19);
+  low = dynCall('jj', ptr, [0x12345678, 0xabcdef19]);
```

Comment:
Why does this need to change? The former way is in general more efficient / less dynamic.

Does this PR make us stop emitting the specialized ones? If so let's talk about that, it sounds worrying but maybe I'm missing the point.

Reply:
I considered generating JS-side dynCalls, one per signature, but I think that would cost a lot more in code side.

Also, once we switch to table calls everywhere we won't need them because we can just make direct variadic calls:

e.g. Rather than involving the signature we can convert:
```
low = dynCall_jj(ptr, 0x12345678, 0xabcdef19);
```

to:

```
low = table.get(ptr)(0x12345678, 0xabcdef19);
```

This is the optimal case with no need to consider the signature and no need to use Array.slice or generate any garbage.  However doing it this way requires more refactoring of `makeDynCall` because the lecacy dynCalls expect ptr to be in the arg0 position, so there is not a pattern that works for both methods.

Reply:
Iiuc `table.get(ptr)(0x12345678, 0xabcdef19);` will require WebAssembly VM support? So users targeting wide set of Wasm browsers will still have to rely on `dynCall_jj`s?

Reply:
When there are lots of signatures, a `dynCall('jj', ...)` would certainly look smaller, but it is also **several** orders of magnitude slower than a `dynCall_jj(ptr, ...)`. Also, there is an issue that if the programs has only very few signatures, then `dynCall('jj', ...)` variant will actually generate bigger resulting program compared to having the few dynCall_jj(ptr, ...)`s listed out.

Given the massive performance drawback, we should only consider `dynCall('jj', ...)` on `-Oz` builds, but even there, only when we can somehow prove/calculate that it will be smaller than the potentially few `dynCall_jj(ptr, ...)`s?

Reply:
> Iiuc table.get(ptr)(0x12345678, 0xabcdef19); will require WebAssembly VM support? 

No, that only depends on basic Table support in the wasm MVP. There should not be compatibility issues.

I agree though that dynamic dynCall, with a string signature `dynCall('..'` is best avoided, though, for performance reasons.

Perhaps we can move straight to `table.get(ptr)(..)`? Or maybe the safest incremental step is to create one dynCall per signature in the JS, that is, to just move where those are defined from wasm (where before this PR we have one per signature) to JS.

However, if using a dynamic dynCall with a string is a useful incremental step towards a very near target of one of those, I'm fine with it, it all doesn't need to be in one PR.

While thinking about the long term goal here, btw, please measure the speed of `table.get(ptr)(..)`. I don't expect problems there, but in the very earliest days of wasm that was not as fast as it should be, so good to verify.